tests/admin_inlines/models.py

```diff
@@ -160,8 +160,8 @@ class OutfitItem(models.Model):
 
 
 class Fashionista(models.Model):
-    person = models.OneToOneField(Person, models.CASCADE, primary_key=True)
-    weaknesses = models.ManyToManyField(OutfitItem, through='ShoppingWeakness', blank=True)
+    person = models.ForeignKey(Person, models.CASCADE)
```

Comment:
It might be that we can't do this. See main comment.

Reply:
I couldn't find a model that is suitable. There are only two models with `ManyToManyField` and both would need changing for these tests. I don't think the added effort of checking these changes is worth the effort.